modules/calypsoify/class.jetpack-calypsoify.php

```diff
@@ -226,21 +226,27 @@ public function remove_core_menus() {
 		remove_submenu_page( 'options-general.php', 'sharing' );
 	}
 
-	public function add_plugin_menus() {
+	public function add_custom_menus() {
 		global $menu, $submenu;
 
-		add_menu_page( __( 'Manage Plugins', 'jetpack' ), __( 'Manage Plugins', 'jetpack' ), 'activate_plugins', 'plugins.php', '', $this->installed_plugins_icon(), 1 );
-
-		// // Count the settings page submenus, if it's zero then don't show this.
-		if ( empty( $submenu['options-general.php'] ) ) {
+		if ( $_GET[ 'post_type' ] == 'feedback' ) {
```

Comment:
I'd recommend using Yoda conditions and strict comparisons when possible.

```suggestion
		if ( 'feedback' === $_GET['post_type'] ) {
```